main.py

```diff
@@ -30,14 +29,24 @@ def main():
             "Exit",
         ]
 
-        for idx, menu in enumerate(menues):
+        for idx, menu in enumerate(menus):
             print(f"{idx}. {menu}")
 
         option = input("Choose an option (0-11): ")
 
         match option:
             case "0":
                 df = import_a_csv_file()
+            case "1":
+                try:
+                    view_all_trans(df)
```

Comment:
```suggestion
                    data_management.view_all_trans(df)
```

You moved `view_all_trans` into `data_management.py`, so you need to add the name of the module of the function.